bin/sl-run.js

```diff
@@ -12,11 +12,19 @@ process.on('disconnect', function() {
   process.exit(2);
 });
 
-var assert = require('assert');
 var config = require('../lib/config'); // May exit, depending on argv
 var log = config.logger;
 var tracer = require('../lib/tracer');
 
+if (process.env.STRONGLOOP_APM_ENABLE) {
+  try {
+    var apmPath = require('../lib/apm')();
+    log.info('APM modules loaded from: %s', apmPath);
+  } catch (err) {
+    // XXX(kev): Do we want to log the loading error?
```

Comment:
Yes, since APM was specifically enabled, but isn't going to happen, we should `log.error("APM failed to load: %s', err.message)` it.